src/main/java/Duke.java

```diff
@@ -1,10 +1,100 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String DOTTED_LINE = "____________________________________\n";
+    private static int numOfTasks = -1;
```

Comment:
Coding violation: plural form should be reserved for variables storing collections of tasks. For an integer, it should be in the singular form.